.stylelintrc.json

```diff
@@ -52,6 +51,24 @@
       {
         "ignorePseudoClasses": ["global"]
       }
-    ]
+    ],
+    "scss/at-import-partial-extension": null,
```

Comment:
New disabled rules will be re-enabled in the follow-up [issue](https://github.com/sourcegraph/stylelint-config/issues/154).